### PR TITLE
Onboarding assisted-events-scrap repository

### PR DESCRIPTION
**Pull request by michaellevy101**

### CODE REVIEW

**Comment by osherdp** [comment]
/uncc @bbguimaraes @hongkailiu
/cc @lranjbar @osherdp 
/hold
(I'll look at it tonight)

File: ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ periodics:
   labels:
     ci-operator.openshift.io/variant: ci-4.1
     ci.openshift.io/generator: prowgen
+    ci.openshift.io/no-builds: "true"
```
Comment (osherdp) [inline ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:15]: sounds weird you'll need to do changes in this scope
maybe you need to rebase?

File: core-services/prow/02_config/openshift-assisted/assisted-events-scrap/_pluginconfig.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift-assisted/assisted-events-scrap
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift-assisted/assisted-events-scrap
+  review_acts_as_lgtm: true
```
Comment (osherdp) [inline core-services/prow/02_config/openshift-assisted/assisted-events-scrap/_pluginconfig.yaml:9]: I think it needs also jira plugin, just like we did for assisted-installer-deployment:
https://github.com/openshift/release/blob/9668b6bb2b2d5c91d46601150bf8c2592529db71/core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_pluginconfig.yaml#L14

File: ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile.build
+  from: coreos_centos_stream8
+  inputs:
+    assisted-service:
+      as:
+      - quay.io/ocpmetal/assisted-service:latest
```
Comment (osherdp) [inline ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml:21]: @lranjbar shouldn't the ``Dockerfile.build`` image use ``quay.io/edge-infrastructure/assisted-service:latest`` instead?
I don't remember if we're updating the old image anymore (or if we want to continue doing that)

File: ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  to: assisted-events-scrap-build
+- dockerfile_path: Dockerfile
+  from: coreos_centos_stream8
+  inputs:
+    assisted-service:
+      as:
+      - quay.io/ocpmetal/assisted-service:latest
```
Comment (osherdp) [inline ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml:28]: same as above

**Review by osherdp — CHANGES_REQUESTED** [review]
looks great, added just few comments

File: ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml (hunk 3 above)
Comment (michaellevy101) [reply ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml:21]: Anyway, now the Dockerfile uses quay.io/ocpmetal/assisted-service:latest ... so for now this is the right path

File: ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml (hunk 4 above)
Comment (michaellevy101) [reply ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml:28]: same as above:)

File: ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml (hunk 3 above)
Comment (lranjbar) [reply ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml:21]: The decision was made awhile back to move away from ocpmetal. This new Dockerfile should use edge-infrastructure and not ocpmetal.

Comment (lranjbar) [reply ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml:21]: I vote 2 PRs and update it now so it is not forgotten about in a week or so.

Comment (osherdp) [reply ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml:21]: yeah let's do this

**Review by osherdp — APPROVED** [review]

**Pull request merged**